Documentation/operations/upgrade.rst

```diff
@@ -318,6 +318,22 @@ Annotations:
   applications create new connections after the TTL specified by the upstream
   DNS server is expired.
 
+New Options
+~~~~~~~~~~~
+
+* ``routing-mode=native``: This option enables native-routing mode, in place of
+  ``tunnel=disabled``, now deprecated.
+* ``tunnel-protocol``: This option allows setting the tunneling protocol, in place
+  of e.g., ``tunnel=vxlan``.
+
+Deprecated Options
+~~~~~~~~~~~~~~~~~~
+
+* The ``tunnel`` option was deprecated and will be removed in v1.14. To enable
+  native-routing mode, you can set ``routing-mode=native`` (previously
+  ``tunnel=disabled``). To configure the tunneling protocol, you can set
```

Comment:
Great example of "to accomplish X, do Y" writing. A small tweak will tighten it up.
```suggestion
  ``tunnel=disabled``). To configure the tunneling protocol, set
```